website/docs/Assertion.md

```diff
@@ -5,6 +5,21 @@ title: Assertion
 
 The [WDIO testrunner](https://webdriver.io/docs/clioptions) comes with a built in assertion library that allows you to make powerful assertions on various aspects of the browser or elements within your (web) application. It extends [Jests Matchers](https://jestjs.io/docs/en/using-matchers) functionality with additional, for e2e testing optimized, matchers, e.g.:
 
+## Soft Assertions
```

Comment:
The location is not ideal, can we move it below of `For the full list, see the...`